src/App.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import "./App.css";
```

Comment:
I think you use styled-components in other files, so please do this here as well.